radix-engine/src/kernel/call_frame.rs

```diff
@@ -100,6 +100,9 @@ pub struct CallFrame {
 
     next_lock_handle: LockHandle,
     locks: HashMap<LockHandle, SubstateLock>,
+
+    /// WASM instance consumed memory for this call frame
+    pub consumed_wasm_memory: usize,
```

Comment:
Can we move this to the module state? It shouldn't be accesssed anywhere other than kernel itself.

Reply:
done